.circleci/config.yml

```diff
@@ -157,10 +157,10 @@ workflows:
               - docs_plugins_common
           filters:
             branches:
-              only: master
+              only: /([0-9\.]*\-build)/
```

Comment:
Notice that if someone will push a change to an older release branch, that will overwrite the more recent one

Reply:
yeah we really need docs with a version select dropdown
make a jira? @mabuaisha

Reply:
@tyacbovi I think for know we can pin it to 5.0.5-build and maybe later on we can update it to dupport multiple versions